app/assets/stylesheets/landing_page/sections/hero.scss

```diff
@@ -14,6 +14,13 @@
   box-shadow: inset 0 0 0 999999px rgba(0, 0, 0, 0.5); /* is there any cleaner way? */
 }
 
+.hero__content {
+  max-width: 1366px;
+  margin: 0 auto;
+  padding: $dimension-mobile-margin;
+  margin-top: -4.44vh; /* -50px, when content height is 900px, 80vh */
```

Comment:
Be sure to test the cross browser behavior of the `vh` unit well: many known issues listed in http://caniuse.com/#feat=viewport-units